src/main/java/lotto/domain/LottoGenerator.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoGenerator {
+    private final int START_INCLUSIVE = 1;
+    private final int END_INCLUSIVE = 45;
+    private final int COUNT = 6;
+    public Lotto generate() {
```

Comment:
```suggestion
    private final int COUNT = 6;
    
    public Lotto generate() {
```
> ### 공백 라인을 의미 있게 사용한다
> 공백 라인을 의미 있게 사용하는 것이 좋아 보이며, 문맥을 분리하는 부분에 사용하는 것이 좋다. 과도한 공백은 다른 개발자에게 의문을 줄 수 있다.

1주차 공통 피드백엔 이런 내용이 있었네요~